stories/setup/App.override.styles.scss

```diff
@@ -0,0 +1,6 @@
+html,
+body {
+  background: #fff;
+  // Giving extra cushion so nothing gets covered.
+  padding: 20px 10px 10px;
```

Comment:
I think it would be nice to follow our app specs to make everything look consistent. Following the specs would mean using pixel values divisible by 2 or 4, etc:

```
padding: 24px 12px 12px;
```

Reply:
oh?! sure thing